modules/administration_manual/pages/configuration/files/encryption/enable-encryption-app.adoc

```diff
@@ -0,0 +1,55 @@
+[[enable-the-encryption-app]]
+= How to Enable the Encryption App
+Matthew Setter <matthew@matthewsetter.com>
+:keywords: encryption, occ
+:description: This guide will show you how to enable the encryption app in ownCloud.
+:experimental:
+:page-partial:
+
+Before you can use encryption you must enable the encryption app. 
+You can do this either from the command-line or from the Web-UI.
+
+== Enable Encryption From the Command-Line
+// tag::enable-encryption-app-via-command-line[]
+To enable the encryption app, run the following command:
+
+[source,console]
+----
+sudo -u www-data php occ app:enable encryption
+----
+
+If the encryption app successfully enables, then you should see the
+following confirmation:
+
+....
+encryption enabled
+....
+
+[NOTE] 
+====
+This should never happen, but the encryption app may not be packaged with your ownCloud installation. 
+If so, you will see the following output when you attempt to enable it:
+
+....
+encryption not found
+....
+
+If that happens, then you need to install it manually. 
+You can do this by cloning the encryption app, using the following command:
+
+[source,console]
+----
+git clone https://github.com/owncloud/encryption.git apps/encryption
```

Comment:
As prerequisite, the admin needs to be in the ownCloud root.
As this is as mentioned not part of the shipped tar, it may go away on the next update.

I would write something like:

Change to the ownCloud root directory of your instance.
Please use in case you have an ``apps-external`` directory this one as installation target otherwise use ``apps``.
If you have an apps directory only, you may need to re-clone the encryption app on the next instance update.